renderdoc/3rdparty/catch/official/catch.hpp

```diff
@@ -1661,7 +1661,14 @@ namespace Catch {
 
 #ifdef CATCH_PLATFORM_MAC
 
-    #define CATCH_TRAP() __asm__("int $3\n" : : ) /* NOLINT */
+    // backported from Catch2
```

Comment:
This is the upstream commit

https://github.com/catchorg/Catch2/commit/bad3c930493d08dd7cca2af57cf355d9b7cddb21